### PR TITLE
create state pages

### PR DESCRIPTION
**Pull request by kofron**

### CODE REVIEW

File: src/components/state-row.js [hunk 1]
```diff
@@ -18,7 +21,7 @@ const StateRow = (data) => {
   if (data.daysToVote < 3) {
     return (
       <tr>
-        <th>{data.state}</th>
+        <th><Link to={statePage}>{data.state}</Link></th>
```
Comment (deanputney) [inline src/components/state-row.js:28]: I'm gonna want to expand this to be on the bar too, but this is a fine start.

File: src/templates/style.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	padding: 1.5rem 1.5rem;
+	line-height: 1.5rem;
+	a {
+		border-bottom: 1px solid;
+		color: black;
+	}
+	a:hover {
+		color: $secondary-color;
+	}
+}
```
Comment (deanputney) [inline src/templates/style.scss:65]: How come all this got changed...?

Comment (deanputney) [reply src/templates/style.scss:65]: Oh I see... Weird that this doesn't come out of `src/components`...

**Review by deanputney — APPROVED** [review]

File: src/templates/style.scss (hunk 2 above)
Comment (kofron) [reply src/templates/style.scss:65]: Yeah, I thought about pulling it from `src/components`, but I figure this lets us change the css independently if we wanted to.  Probably not necessary but it was easy enough.

File: src/components/state-row.js (hunk 1 above)
Comment (kofron) [reply src/components/state-row.js:28]: Agreed.  My attempt at a trivial implementation of that by adding the `Link` to the `td` failed, so I punted.  Let me see if I can use a raw `href` to accomplish this... feels ugly to mix the two but fuck it, I'll give it a shot

Comment (kofron) [reply src/components/state-row.js:28]: Figured it out, and managed to fix the bar width bug in the process.

File: src/templates/style.scss (hunk 2 above)
Comment (deanputney) [reply src/templates/style.scss:65]: Yeah, the problem with that is that it exists in both places then... We should bubble up the component level stuff to the components I guess and include that separately.

**Pull request merged**